examples/index.tsx

```diff
@@ -6,8 +6,15 @@ import * as React from 'react';
 import { ThemeProvider } from '@itwin/itwinui-react';
 
 const withThemeProvider = (Component: () => React.ReactElement) => () => {
+  const [portalContainer, setPortalContainer] = React.useState<HTMLElement>();
+  React.useEffect(() => void setPortalContainer(document.body), []);
```

Comment:
Why is `void` needed?

Reply:
the `void` is to ensure that if there is ever a return a value, it will be discarded.